lms/djangoapps/courseware/rules.py

```
@@ -46,3 +47,15 @@ def query(self, user):
         # that is used to determine if the rule should allow a user
         # into django admin
         return Q(pk__in=[])
+
+
+class HasStaffRolesRule(Rule):
+    """
+    A rule that checks whether a user has one of the following roles in a course:
+    Staff, Instructor, Beta Tester, Forum Community TA, Forum Group Moderator, Forum Moderator, Forum Administrator
+    """
+    def check(self, user, course_key):
+        return has_staff_roles(user, course_key)
+
+    def query(self, user):
+        raise NotImplementedError()
```

Do we want to raise an error here?